### DISCUSSION TITLE
Where I can find a cool platform to study ReactJS?

### DISCUSSION THREAD

**Discussion by RafaelNevesdeOliveira** [Schweinepriester/github-profile-achievements#564] [General]
Hey fellows, where i can find a platform to study ReactJS?

**Comment by ThamirisdaSilva** [comment]
> Hey fellows, where i can find a platform to study ReactJS?

@RafaelNevesdeOliveira at RocketSeat

**Comment by cybersaksham** [comment]
> Hey fellows, where i can find a platform to study ReactJS?

Search `codewithharry` on youtube.
Or you can go with `Angela Yu` complete website BootCamp on udemy.

**Reply by RafaelNevesdeOliveira** [reply]
Tks @cybersaksham

**Comment by mmarifat** [comment]
You can try this Youtube Channel
https://www.youtube.com/channel/UC6vRUjYqDuoUsYsku86Lrsw

**Comment by tauanesales** [comment]
Angela Yu courses are really good. She has a [web development bootcamp](https://www.udemy.com/course/the-complete-web-development-bootcamp/) in which she teaches React.js on Udemy.

**Comment by ArielMAJ** [comment]
There are some Coursera Meta's courses in which they teach React/ReactJS/ReactNative. They look like they are good courses. Coursera courses usually are pretty great and feel like you're taking college classes. They usually push you to actually learn.

- [Web front-end](https://www.coursera.org/professional-certificates/meta-front-end-developer)
- [Android](https://www.coursera.org/professional-certificates/meta-android-developer)
- [iOS](https://www.coursera.org/professional-certificates/meta-ios-developer)
- [Back-end](https://www.coursera.org/professional-certificates/meta-back-end-developer)

**Comment by HasanSafarli** [comment]
codewithharry on Youtube!

**Comment by Sharique55** [comment]
Hi @RafaelNevesdeOliveira 

---

# 🔥 Top Platforms to Learn **ReactJS** – Curated for Developers in 2025 🚀

Want to master ReactJS and build blazing-fast UIs? Here’s a **handpicked list of platforms** that offer the best learning paths, from beginner to advanced — **with proper links, free + paid options, and community support!** 🌍💡

---

## 🧑‍💻 1. **React Official Docs**

The best place to start your journey. Updated, interactive, and written by the React team.
📘 Learn directly from the source.

🔗 [https://react.dev/learn](https://react.dev/learn)

> 💬 *“You should always trust the official docs first.”*

---

## 📹 2. **freeCodeCamp – Full React Course (YouTube)**

💯 Completely free, beginner-friendly, and covers hooks, props, state, and more.

🔗 [https://www.youtube.com/watch?v=bMknfKXIFA8](https://www.youtube.com/watch?v=bMknfKXIFA8)

> 📺 *Over 6 hours of pure React awesomeness!*

---

## 💼 3. **Scrimba – Learn React for Free**

A **highly interactive platform**. It mixes screencasts with live coding in the browser!
Perfect for beginners + career switchers.

🔗 [https://scrimba.com/learn/learnreact](https://scrimba.com/learn/learnreact)

> ⚡ *Best if you love hands-on practice while watching.*

---

## 📚 4. **Epic React by Kent C. Dodds**

An advanced and **premium-level course** for serious React developers.
Built by a well-known industry expert.

🔗 [https://epicreact.dev/](https://epicreact.dev/)

> 🔐 *Paid but totally worth the investment for senior roles.*

---

## 🎯 5. **Frontend Masters – React Path**

Top-tier instructors like Brian Holt and Steve Kinney teach React here.
It includes **real-world practices**, TypeScript, performance, and testing.

🔗 [https://frontendmasters.com/learn/react/](https://frontendmasters.com/learn/react/)

> 🧠 *Great for intermediate to advanced devs.*

---

## 🧠 6. **React Projects on GitHub**

Real code, real issues, real learning.
Explore, fork, contribute — learn by doing! 🤝

🔗 [https://github.com/topics/react](https://github.com/topics/react)
🔗 [https://github.com/enaqx/awesome-react](https://github.com/enaqx/awesome-react)

> 💥 *Contributing to open-source = next level React mastery.*

---

## 🎓 7. **Codecademy – Learn React**

An interactive course platform with quizzes and projects. Great UI and gamified experience.
Beginner to Intermediate.

🔗 [https://www.codecademy.com/learn/react-101](https://www.codecademy.com/learn/react-101)

> 🎮 *Feels like playing a dev game!*

---

## 📕 8. **ReactJS Handbook by Flavio Copes**

Short and practical. Free to read online. Covers the core concepts crisply.
Great as a reference book too!

🔗 [https://www.freecodecamp.org/news/the-react-handbook-b71c27b0a795/](https://www.freecodecamp.org/news/the-react-handbook-b71c27b0a795/)

> 📓 *Best for quick refreshers.*

---

## 🤹‍♀️  **Practice Platforms**

These sites let you **build projects and get feedback**:

* 🌐 [https://www.frontendmentor.io/](https://www.frontendmentor.io/)
* 🧪 [https://codewell.cc/](https://codewell.cc/)
* 📦 [https://reactplay.io/](https://reactplay.io/)

---
